js/renovation/ui/common/widget.tsx

```diff
@@ -112,7 +119,8 @@ export class Widget extends JSXComponent(WidgetProps) {
   widgetRef!: HTMLDivElement;
 
   @Effect()
-  accessKeyEffect() {
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  accessKeyEffect(): any {
```

Comment:
Maybe we can use
`(() => void) | undefined`

Also, we can try replace `return undefined` by `return noop` and get type `() => void`

Reply:
fixed